style/style.css

```diff
@@ -0,0 +1,99 @@
+main {
+  display: grid;
+  grid-template-rows: 10fr 4fr 2fr;
+}
+
+a {
+  text-decoration: none;
+  color: rgb(0, 136, 204);
+}
+
+.top-section {
+  display: flex;
+  flex-flow: column nowrap;
+  justify-content: space-evenly;
+  align-items: center;
+  height: 500px;
+  vertical-align: middle;
+  background: url(/Backgrounds-and-Gradients/images/background1.jpg) 35% 90%;
+}
+
+.top-section h1,
+h2 {
+  height: 15%;
+  color: #fff;
+}
+
+.play {
+  width: 75%;
+  background: url(/Backgrounds-and-Gradients/images/play-button.png) 100% 0 no-repeat;
+}
+
+.nav-input-wrap {
+  display: inline-block;
+  text-align: center;
+}
+
+.nav-bar {
+  display: flex;
+  width: 90%;
+  border-radius: 15px;
+  background: -webkit-gradient(linear, 0 0, 0 100%, from(rgba(0, 0, 0, 0.1)), color-stop(0.05, rgba(0, 0, 0, 0)), color-stop(0.97, rgba(0, 0, 0, 0)), to(rgba(0, 0, 0, 0.3))), -webkit-gradient(linear, 0 0, 100% 0, from(rgba(0, 0, 0, 0.8)), color-stop(0.003, rgba(0, 0, 0, 0.25)), color-stop(0.998, rgba(0, 0, 0, 0.25)), to(rgba(0, 0, 0, 0.25))) no-repeat 50% 50%, -webkit-gradient(linear, 0 100%, 0 0, from(#666), color-stop(0.5, #5e5e5e), color-stop(0.51, #979292), to(#979292));
+}
+
+.nav-bar button {
+  width: 12%;
+  height: 40px;
+  border: 1px solid silver;
+  background: transparent;
+  color: #fff;
+}
+
+.nav-bar-image {
+  background: center url(/Backgrounds-and-Gradients/images/apple-logo.png) no-repeat;
+}
+
+input[type="search"] {
+  width: 80%;
+  margin-top: 8px;
+  border-radius: 15px;
+  background: url(/Backgrounds-and-Gradients/images/magnifying-glass.png) 10% no-repeat;
+}
+
+.image-tile {
+  display: flex;
+}
+
+.image-tile * {
+  width: 100%;
+  height: 220px;
+}
+
+.image-tile h1:nth-child(1) {
+  background: url(/Backgrounds-and-Gradients/images/background2.jpg) center no-repeat;
```

Comment:
The correct syntax here should be something like `background: url(../images/background2.jpg) center no-repeat;`. Kindly check and update throughout.

Reply:
Thanks alot i have done that.